master-nonlatent/master.cfg

```diff
@@ -223,10 +227,6 @@ f_dockerlibrary.addStep(steps.ShellCommand(
     command=["bash", "-xc", util.Interpolate("./docker-library-test.sh \"%(prop:tarbuildnum)s\" \"%(prop:parentbuildername)s\"")]))
 f_dockerlibrary.addStep(steps.Trigger(name='WordPress', schedulerNames=['s_wordpress'], waitForFinish=False, updateSourceStamp=False,
     set_properties={"tarbuildnum": Property("tarbuildnum")}, doStepIf=lambda step: str(step.getProperty("parentbuildername")).startswith('amd64')))
-f_dockerlibrary.addStep(steps.ShellCommand(
-    name="cleanup test image",
-    command=["buildah", "rmi", util.Interpolate("mariadb-%(prop:tarbuildnum)s-%(prop:parentbuildername)s")],
-    doStepIf=lambda step: not savePackage(step)))
```

Comment:
Shouldn't this be kept for cases when Wordpress is not triggered?

Reply:
I thought this would remove the image/tag before wordpress ran the tests
.
I'll push that the tag is generated for amd64, even if we're doing that test/condition in two places.

Reply:
Yes it will. But Wordpress is only triggered for amd64. So for the other arches shouldn't the cleanup step run? Something like having a doStepIf=lambda x: not x.startswith("amd64")

Reply:
handled by [docker-library-build.sh](https://github.com/MariaDB/buildbot/pull/58/files#diff-6f3f6e52310fc1d8040e5a1d91800d6bca939772599f2912a8dfbfde7b01fcc6) which needs to exist otherwise we create the same tag with two different arches in a race condition with wordpress.